eks-anywhere-snow/Addons/Partner/Sysdig/sysdig.yaml

```diff
@@ -0,0 +1,43 @@
+---
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  name: sysdig
+  namespace: sysdig
+spec:
+  chart:
+    spec:
+      chart: sysdig-deploy
+      reconcileStrategy: ChartVersion
+      sourceRef:
+        kind: HelmRepository
+        name: sysdig
+        namespace: flux-system
+      version: 1.5.46
+  interval: 30s
+  url: https://charts.sysdig.com
+  values:
+    global:
+      sysdig:
+        accessKey: ${sysdigAccessKey}
```

Comment:
Did it work for your locally to pull your Secrets for `sysdig-accesskey` AWS Secrets Manager?